scripts/bert/create_pretraining_data.py

```diff
@@ -26,36 +26,43 @@
 import glob
 import collections
 import random
+import time
+from multiprocessing import Pool
+import numpy as np
 import mxnet as mx
 import gluonnlp as nlp
-
 from gluonnlp.data import BERTTokenizer
 
 
-logging.getLogger().setLevel(logging.DEBUG)
-
 parser = argparse.ArgumentParser(
     description='Pre-training data generator for BERT')
 
-parser.add_argument('--input_file', type=str, default=None, help='input_file')
-
 parser.add_argument(
-    '--output_file',
+    '--input_file',
     type=str,
     default=None,
-    help='Output MXRecordIO example file (or comma-separated list of files).')
+    help='input file(s). For example, "~/data/*.txt"')
 
 parser.add_argument(
-    '--vocab_file',
+    '--output_dir',
     type=str,
     default=None,
-    help='The vocabulary file that the BERT model was trained on.')
+    help='Output directory.')
 
 parser.add_argument(
-    '--do_lower_case',
-    action='store_true',
-    help='Whether to lower case the input text. Should be True for uncased'
-    'models and False for cased models.')
+    '--format',
+    type=str,
+    default='numpy',
+    choices=['numpy', 'recordio'],
+    help='Output file format. If set to "numpy", examples are serialized as a single "npz" file.'
+         'If set to "recordio", examples are serialized using IndexedRecordIO format.')
+
+parser.add_argument(
+    '--vocab',
+    type=str,
+    default=None,
+    help='The dataset name for the vocab file BERT model was trained on. For example, '
+         '"book_corpus_wiki_en_uncased"')
```

Comment:
what are the options? where can they be found

Reply:
options added